concept/type/impl/ThingTypeImpl.java

```diff
@@ -79,13 +80,14 @@ void setSupertype(ThingType thingType) {
 
         @Override
         public ThingTypeImpl getSupertype() {
-            final TypeImpl supertype = super.getSupertype();
+            TypeImpl supertype = super.getSupertype();
```

Comment:
Why did we lose `final` here?
```suggestion
            final TypeImpl supertype = super.getSupertype();
```

Reply:
Good catch. I missed them because we never use `final` on local variables in other repositories. Fixed in https://github.com/graknlabs/client-java/pull/211/commits/63040ad062025b93a3389394149933c6e99bf1c4